### PR TITLE
Improve Docker PHP Variables / remove hardcoded folders

### PR DESCRIPTION
**Pull request by pabloguerino**
For issue #26 

added more details in the comments on that issue

### CODE REVIEW

**Comment by jbguerraz** [comment]
LGTM

File: Makefile [hunk 1]
```diff
@@ -3,6 +3,14 @@ $(shell cp -n \.env.default \.env)
 $(shell cp -n \.\/src\/docker\/docker-compose\.override\.yml\.default \.\/src\/docker\/docker-compose\.override\.yml)
 include .env
 
+# Setup PHP Variables based on package version
+PHP_IMAGE=php:$(PHP_VERSION)-fpm
```
Comment (jbguerraz) [inline Makefile]: Probably great to remove the dot in PHP_VERSION here

Comment (pabloguerino) [inline Makefile]: Updated!

File: Makefile [hunk 2]
```diff
@@ -3,6 +3,15 @@ $(shell cp -n \.env.default \.env)
 $(shell cp -n \.\/src\/docker\/docker-compose\.override\.yml\.default \.\/src\/docker\/docker-compose\.override\.yml)
 include .env
 
+# Setup PHP Variables based on package version
+
+PHP_IMAGE=php:$(shell printf '%s' "$(PHP_VERSION)" | sed -e 's/\.//')-fpm
+PHP_CONFIG=/etc/php$(PHP_VERSION)
```
Comment (andriyun) [inline Makefile]: I think PHP_CONFIG  var is useless
we can use /etc/php${PHP_VERSION} in docker-compose.override.yml.default

Comment (gaydabura) [inline Makefile]: https://github.com/skilld-labs/skilld-docker-container/pull/31/files#diff-18c84fa0272bccd61d728cfcb98224c0R8
its included. )

Comment (gaydabura) [inline Makefile]: See this https://github.com/skilld-labs/skilld-docker-container/pull/31#discussion_r107454428

**Comment by pabloguerino** [comment]
Updated PR to match last comments, thx for the review!

**Pull request merged**